src/index.js

```diff
@@ -168,21 +169,40 @@ class StickyTable extends React.Component {
       ['leftColumn', 'width', 'leftStickyColumnCount'],
       ['rightColumn', 'width', 'rightStickyColumnCount']
     ].forEach(([stickyKey, sizeKey, countPropKey]) => {
-      var insets = [0];
-      var count = props[countPropKey];
-      var netInset = 0;
-
-      //HINT we only want this loop for the second sticky and up
-      for (s = 1; s < count; s++) {
-        var node = stickyKey === 'header' || stickyKey === 'leftColumn' ? cellNodes[0] : cellNodes[cellNodes.length - 1];
-
-        if (node) {
-          var boundingRect = node.getBoundingClientRect();
-
-          netInset += boundingRect[sizeKey];
+      var insets = [];
+
+      if (props[countPropKey] > 1) {
+        insets = [0]
+        var count = props[countPropKey];
+        var netInset = 0;
+
+        // HINT we only want this loop for the second sticky and up
+        for (s = 1; s < count; s++) {
+          var node = undefined;
+
+          switch (stickyKey) {
+            case 'header':
+              node = rowNodes[s - 1]
+              break;
+            case 'footer':
+              node = rowNodes[rowNodes.length - s]
```

Comment:
when the node is a row, we should be using childNodes[0] so we're always getting size of a cell - in case there's a browser bug with table row height